### PR TITLE
.ping issue

### PR DESCRIPTION
**Pull request by zaxary**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -99,6 +99,21 @@ client.on("message", async message => {
     'Outlook not so good',
     'Very doubtful'
   ];
+//!ping to run ping
+client.on('message', message => {
+if (message.author.bot) return;
+if (!message.content.startsWith('!')) return;
```
Comment (timot3) [inline index.js]: This will trigger the Rhythm bot as well.

File: index.js [hunk 2]
```diff
@@ -267,9 +283,6 @@ client.on("message", async message => {
       message.channel.send(reminders[rand]);
     } else if (command.match(/\blofi\b/) != null) {
       message.channel.send("https://open.spotify.com/playlist/1DcvziAZBZk1Ji1c65ePtk?si=Qtvu64zsQQurDtQa60tPBg");
-    } else if (command.match(/\bping\b/) != null) {
```
Comment (timot3) [inline index.js:270]: Why not send the embed in this snippet here??
`message.channel.send({embed: {
        color: 0x2ed32e,
        fields: [{
            name: "Your Ping",
            value: "My Ping: " + Math.round(client.ws.ping) + ' ms'
      }
     ],
}`

File: index.js (hunk 1 above)
Comment (zaxary) [reply index.js]: ye my bad I forgot it should be queen ping. Here is the updated code

client.on('message', message => {
if (message.author.bot) return;
if (!message.content.startsWith('queen')) return;
if (message.content.startsWith('queen' + 'ping')) {
    message.channel.send({embed: {
        color: 0x2ed32e,
        fields: [{
            name: "Your Ping",
            value: "My Ping: " + Math.round(client.ws.ping) + ' ms'
      }
     ],
}
})
}})

File: index.js (hunk 2 above)
Comment (zaxary) [reply index.js:270]: yup thats a better idea lmao

File: index.js (hunk 1 above)
Comment (s-hfarooq) [reply index.js]: You also can't have a client.on() within another client.on()

Comment (zaxary) [reply index.js]: ye lets reject my pull request i'll resubmit it again lol

**Pull request closed without merging**